### PR TITLE
Newmycv

### PR DESCRIPTION
**Pull request by ShuryginaA**

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
```
Comment (allnes) [inline modules/0_git/test/main.cpp:10]: problem isn't solved

**Comment by dkurt** [comment]
Please stop create new PRs - let's chat and discuss how to make it properly

**Comment by ShuryginaA** [comment]
> Please stop create new PRs - let's chat and discuss how to make it properly

Yes, I do not understand what is wrong

**Comment by dkurt** [comment]
@ShuryginaA, In Discord please

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/106

**Pull request closed without merging**